app/src/main/AndroidManifest.xml

```diff
@@ -31,8 +32,6 @@
         <activity android:name=".MeetingsActivity" />
         <activity android:name=".AnnouncementActivity" />
         <activity android:name=".ShoppingActivity" />
-        <activity android:name=".EditDeleteTodo" />
```

Comment:
Hey, you need to add these stuff back or you won't be able to access those activities.